delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/DelegatedAccessApplication.properties

```diff
@@ -30,9 +30,9 @@ noDelegatedAccessUser=This user does not have any delegated access
 noHierarchy=The hierarchy hasn't been setup for this tool.  In order to use Delegated Access, you must create a hierarchy.  Please consult the following confluence page: https://confluence.sakaiproject.org/display/DAC/Delegated+Access+Tool
 addAccess=Add Access
 add=Add
-remove=Remove
```

Comment:
Can you revert these changes? You're changing the default locale.

Reply:
double key value

Reply:
Good